translations.csv

```diff
@@ -182,7 +182,7 @@ SCROLL_ADVENTURE_SHORT,[color=white]Contains information about advanced movement
 DIAG_DEFAULT,[color=white]I am an NPC.[/color],[color=white]Ich bin ein NSC.[/color]
 DIAG_START0,[color=white]Welcome to [color=red]the Lost Catacombs[/color]. You can rest here and travel between worlds. You may also exchange [color=yellow]gold[/color] you have collected so far for items here as well. I can move these catacombs to different points around the world. Please talk to me again to see a map of all these points. I can take you there with my magic.[/color],"[color=white]Wilkommen zu [color=red]der verlorene Gruft[/color]. Sie können hier ausrühen und zwischen Welte reisen. Sie durfen auch Ihre [color=yellow]Gold[/color] für Gegenstände austauchen. Ich kann diese Gruft zur verschiedene Punkte um die Welt umziehen. Bitte mit mir wieder sprechen, falls Sie eine Karte von diese Punkte sehen wollen.[/color]"
```

Comment:
I think that last sentence in the English one should actually belong to the dialog below. So just remove it from there and replace it below.

Reply:
Right! I do that. I had troubles when trying to launch the game (also before modifying the translations.csv file), I'll open an issue